src/content/docs/en/guides/images.mdx

```diff
@@ -91,191 +91,17 @@ import myImage from '../assets/my_image.png'; // Image is 1600x900
 
 #### Properties
 
-##### src (required)
+- src (required)
+- alt (required)
+- width and height (required for `public/` and remote images)
+- format
+- quality
+- densities
+- widths
```

Comment:
It feels weird to still include this “reference-y” if brief list of props here (and the same for `<Picture>` below). In general, although somme stuff has been moved, both thes `<Image>` and `<Picture>` sections still feel like reference entries to me: “This component exists. It does X. Code snippet. There are props.”

Is there a world where these get rewritten to follow a guide structure?

Like thinking use case-forward instead of API-forward something along the lines of this (assuming breaking up by component type still makes sense):

```diff
- ### `<Image />`
+ ### Display optimized images with the `<Image />` component

...

- ### `<Picture />`
+ ### Create responsive images with the `<Picture />` component

...

- ### `<img>`
+ ### Display unprocessed images with the HTML `<img>` tag
```

Reply:
ooh, that looks lovely. OK, let's work that in!